obelisk_ws/src/robots/achilles_model/mujoco/achilles.xml

```diff
@@ -0,0 +1,321 @@
+<!-- <?xml version="1.0" encoding="utf-8"?> -->
+<mujoco model="achilles">
+
+  <!-- Set the angles to be in radians -->
+  <compiler angle="radian" autolimits="true" eulerseq="zyx" meshdir="./meshes"/>
+
+  <!-- Fix simulation options -->
+  <option timestep="0.002" integrator = "RK4"/>
+
+  <!-- Uncomment the next line to enable only select contacts. Specifically, this sets the default to no collision. -->
+  <!-- <default> <geom contype="0" conaffinity="0"/> </default> -->
+
+  <!-- Specify the light settings -->
+  <visual>
+      <headlight  diffuse="0.6 0.6 0.6" ambient="0.3 0.3 0.3" specular="0 0 0"/>
+      <rgba haze="0.15 0.25 0.35 1"/>
+      <global elevation="-35"/>
+  </visual>
+
+  <!-- Default camera location -->
+  <statistic extent="2" center="0 0 0.5"/>
+
+  <!-- List the location of all meshes -->
+  <asset>
+    <texture type="skybox" builtin="gradient" rgb1="0.3 0.5 0.7" rgb2="0 0 0" width="512" height="3072"/>
+
+    <mesh file="torso.STL" name="torso"/>
+
+    <mesh file="left_hip_yaw.STL" name="left_hip_yaw"/>
+    <mesh file="left_hip_roll.STL" name="left_hip_roll"/>
+    <mesh file="left_hip_pitch.STL" name="left_hip_pitch"/>
+    <mesh file="left_knee_pitch.STL" name="left_knee_pitch"/>
+
+    <mesh file="left_shoulder_pitch.STL" name="left_shoulder_pitch"/>
+    <mesh file="left_shoulder_roll.STL" name="left_shoulder_roll"/>
+    <mesh file="left_shoulder_yaw.STL" name="left_shoulder_yaw"/>
+    <mesh file="left_elbow_pitch.STL" name="left_elbow_pitch"/>
+
+    <mesh file="right_hip_yaw.STL" name="right_hip_yaw"/>
+    <mesh file="right_hip_roll.STL" name="right_hip_roll"/>
+    <mesh file="right_hip_pitch.STL" name="right_hip_pitch"/>
+    <mesh file="right_knee_pitch.STL" name="right_knee_pitch"/>
+
+    <mesh file="right_shoulder_pitch.STL" name="right_shoulder_pitch"/>
+    <mesh file="right_shoulder_roll.STL" name="right_shoulder_roll"/>
+    <mesh file="right_shoulder_yaw.STL" name="right_shoulder_yaw"/>
+    <mesh file="right_elbow_pitch.STL" name="right_elbow_pitch"/>
+
+    <mesh file="ankle_pitch.STL" name="ankle_pitch"/>
+
+
+    <texture type="2d" name="plane" builtin="checker" mark="edge" rgb1="0.2 0.3 0.4" rgb2="0.1 0.2 0.3"
+      markrgb="0.8 0.8 0.8" width="300" height="300"/>
+    <material name="plane" texture="plane" texuniform="true" texrepeat="5 5" reflectance="0.2"/>
```

Comment:
nit: i'd probably group these with the skybox texture. i'd also separate these non-mesh assets with a comment from the mesh assets (which should also have a comment)